.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: csharp
+solution: MoreLinq.sln
+mono: latest
+dist: trusty
+sudo: required
+dotnet: 1.0.0-preview2-003131
+
+install:
+  - dotnet restore
+
+script:
+  - dotnet build -c Debug MoreLinq
```

Comment:
> > dotnet insisted on trying to use the debug build.
>
>Really? :open_mouth: I guess we could leave it as a separate issue to be investigated?

Hmm. This works on my local machine but did not when I tried on travis. Let's try again to rule out typos. Otherwise it would look like a tooling bug (as you can see from the yml file, the tooling version is not the latest one).